sdk/netapp/Azure.ResourceManager.NetApp/tests/ResourceTests/ANFBackupPolicyTests.cs

```diff
@@ -176,7 +171,7 @@ public async Task CreateVolumeWithBackupPolicy()
             {
                 DefaultVirtualNetwork = await CreateVirtualNetwork();
             }
-            NetAppVolumeBackupConfiguration backupPolicyProperties = new(backupPolicyResource1.Id, false, vault.Id, true);
+            NetAppVolumeBackupConfiguration backupPolicyProperties = new(backupPolicyResource1.Id, false, true);
```

Comment:
I just noticed this, are we using the internal constructors here?
We should not using internal constructors in test cases despite we could.
This might make things different from it is used by our customers who cannot use internal members.
And using internal constructors makes you hard to maintain your test cases because every time you introduce a new property, you might have to change your implementation despite it is not a breaking change